awx/main/models/base.py

```diff
@@ -61,7 +60,10 @@
     (PERM_INVENTORY_CHECK, _('Check')),
 ]
 
-CLOUD_INVENTORY_SOURCES = list(CLOUD_PROVIDERS) + ['scm']
+
+def discover_cloud_inventory_sources():
```

Comment:
add a docstring and type annotations?

Reply:
Thank you! I have added those here: https://github.com/ansible/awx/pull/15537/commits/29c3b8b9b814ed7f7f1b3679e4a921761df20caa

I will leave this comment unresolved in case there's additional feedback.

Reply:
@djyasin that's just a docstring. Type annotations go into function signatures. Here's how: https://github.com/ansible/awx/pull/15537/files#r1786985436.

Reply:
Thank you so much! I have committed your suggestion.